### PR TITLE
Add Deadbolt v0.1.0

### PR DESCRIPTION
**Pull request by alichtman**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/deadbolt.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'deadbolt' do
+  version '0.1.0'
+  sha256 'c302f1c532082b200d844d17cd8feadc61821d97591fb5a0dbb1bb97c3a5981b'
+
+  url "https://github.com/alichtman/deadbolt/releases/download/#{version}/Deadbolt-#{version}.dmg"
+  appcast 'https://github.com/alichtman/deadbolt/releases.atom'
+  name 'Deadbolt'
+  homepage 'https://github.com/alichtman/deadbolt'
+
+  app 'Deadbolt.app'
```
Comment (vladimyr) [inline Casks/deadbolt.rb:10]:
```suggestion
  app 'Deadbolt.app'
  
  zap trash: [
               '~/Library/Application Support/deadbolt',
               '~/Library/Preferences/org.alichtman.deadbolt.plist',
               '~/Library/Saved Application State/org.alichtman.deadbolt.savedState',
             ]
```

**Review by vladimyr — COMMENTED** [review]
I believe it is probably too early to submit it here but nevertheless nice work! 👍

**Comment by alichtman** [comment]
Thanks for the edits! Although it looks like this project was released for the first time yesterday, it was actually released multiple times previously under a different name: “QuickLock macOS. As it’s been expanded to Linux and Windows, it’s been rebranded, and now we have “Deadbolt”.

**Comment by vladimyr** [comment]
@alichtman Sorry to bug you but please remove the extra blank line and trailing whitespace. I absolutely hate this suggestion thing, mailing patches was stellar technology compared to this 😩

**Comment by alichtman** [comment]
@vladimyr `$ brew cask style --fix deadbolt.rb` reports no fixes needed. Does the syntax look good to you now?

**Comment by vladimyr** [comment]
> Thanks for the edits! Although it looks like this project was released for the first time yesterday, it was actually released multiple times previously under a different name: “QuickLock macOS”. As it’s been expanded to Linux and Windows, it’s been rebranded, and now we have “Deadbolt”.

Honestly, I haven't even noticed the release date. I was worried it might [not be notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):
> 
> * The Cask is too obscure. Examples:
>   
>   * A self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 50 stars).
>   * [Electronic Identification (eID) software](https://github.com/Homebrew/homebrew-cask/issues/59021). 

Technically you are over the threshold but we'll see what @vitorgalvao says about that.

**Comment by vladimyr** [comment]
@alichtman All checks are green, we are good now 👍

**Comment by alichtman** [comment]
Great, I'll check back on this issue whenever @vitorgalvao has a moment to look at it. Thanks for the help, @vladimyr!

**Comment by alichtman** [comment]
Thanks!

**Comment by vladimyr** [comment]
Glad I could help, keep up the good work!

**Pull request merged**